exec/bytecode/defer.go

```diff
@@ -0,0 +1,78 @@
+/*
+ Copyright 2020 The GoPlus Authors (goplus.org)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+// Package bytecode implements a bytecode backend for the Go+ language.
+package bytecode
+
+import (
+	"github.com/goplus/gop/exec.spec"
+)
+
+type theDefer struct {
+	next *theDefer
+	Stack
```

Comment:
why need a stack?

Reply:
Save the return value of the function in the defer function parameter.  example:

```
package main

import (
	"fmt"
)

func main() {
	defer fmt.Println(fmt.Println("Hello, playground"))
	fmt.Println("start")
}
```

Saved stack output:

```
Hello, playground
start
18 <nil>
```

The stack output is not saved：

```
Hello, playground
start
7 <nil>
```

Reply:
I see. It really works, but may have a better solution.
I will merge it at first.

Reply:
Ok, look forward to a better solution